app/components/AllOrders.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+export class AllOrders extends React.Component {
```

Comment:
maybe add comment: "if in the end this component is not a smart component, change this from a `class` to a `function` object"